test/built-ins/Proxy/enumerate/return-trap-result.js

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) 2015 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 9.5.11
+description: >
+    [[Enumerate]] ()
+
+    11. Return trapResult
+---*/
+
+var target = {
+    attr: 1
+};
+var foo = { bar: 1 };
+var p = new Proxy(target, {
+    enumerate: function() {
+        return ["a", "b", "c"][Symbol.iterator]();
+    }
+});
+
+var results = [];
+for (let x in p) {
+    results.push(x);
+}
+
+assert.sameValue(results, ["a", "b", "c"]);
```

Comment:
`assert.sameValue` cannot be used here. Change to compareArray (from harness/compareArray.js).